demo/site/src/common/components/Breadcrumbs.sc.ts

```diff
@@ -5,7 +5,7 @@ export const Container = styled.div`
     grid-column: 2 / 24;
 `;
 
-export const Link = styled.a`
+export const Link = styled.span`
```

Comment:
This styled-component should wrap the Next link: `const Link = styled(NextLink)`. Otherwise color and text-decoration both won't work.

Reply:
true 👍  

--> done